src/General/Icon/Icon.test.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+import * as renderer from 'react-test-renderer';
+import { fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import Icon from './Icon';
+
+const testId = 'icon-svg';
+const children = (
+  <path d="M100 56.563H56.562V100H43.438V56.562H0V43.438h43.438V0h13.124v43.438H100z"></path>
+);
+
+it('<Icon> should render a svg with color, width, height and a path as children', () => {
```

Comment:
suggestion: 
1. replace all the `it` with `test`
2. change the description of this case to `it matches snapshot`

Reply:
can you render the snapshot without `children`?

Reply:
hmm, can `<Icon />` be rendered without children in all the test cases?

Reply:
Nope, because `children` is required prop for `Icon`.
Got an error `Property 'children' is missing in type '{}' but required in type 'Props'`

Reply:
ok